styles/Canonical/003-Ubuntu-names-versions.yml

```
@@ -13,11 +13,11 @@ scope:
 level: suggestion
 tokens:
   # LTS releases should be followed by "LTS" suffix.
-  - 'Ubuntu (24|22|20|18|16|14|12|10|8)\.04(?! LTS)'
+  - 'Ubuntu (\d)?(0|2|4|6|8)\.04(?! LTS)'
```

More classic regexp

```suggestion
  - 'Ubuntu \d?[02468]\.04(?! LTS)'
```

But maybe you want this

```suggestion
  - 'Ubuntu [1-9]?[02468]\.04(?! LTS)'
```